repos/fdbt-site/src/pages/searchOperators.tsx

```diff
@@ -142,31 +170,62 @@ export const renderSearchBox = (operatorsAdded: boolean, errors: ErrorInfo[]): R
     );
 };
 
-export const showSearchResults = (searchText: string, searchResults: Operator[], errors: ErrorInfo[]): ReactElement => {
+export const showSearchResults = (
+    searchText: string,
+    searchResults: Operator[],
+    errors: ErrorInfo[],
+    selectedOperators: Operator[],
+    setSelectedOperators: React.Dispatch<React.SetStateAction<Operator[]>>,
+): ReactElement => {
     const addOperatorsErrors: ErrorInfo[] = [];
     errors.forEach((err) => {
         if (err.id === addOperatorsErrorId) {
             addOperatorsErrors.push(err);
         }
     });
+    const [searchResultsCount, setSearchResultsCount] = useState(searchResults.length);
+    const [showSearchResultsLine, setShowSeearchResultsLine] = useState(true);
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const formattedoperatorToAdd = [event.target.value].map((item: string) => ({
```

Comment:
Is it needed to map over this and create it as an array, only for it to be spread back into an object again? 

e.g.

const operatorToAdd = event.target.value;
const formattedOperatorToAdd = { nocCode:  operatorToAdd.split('#')[0], name:  operatorToAdd.split('#')[1] }